package.json

```diff
@@ -9,7 +9,8 @@
     "test": "jest",
     "coverage": "jest --collect-coverage",
     "lint": "eslint .",
-    "lint-fix": "eslint --fix ."
+    "lint-fix": "eslint --fix .",
+    "prepare": "husky install"
```

Comment:
Should this be done as a post install script instead so it does not have to be done manually?

Reply:
Good call, i'll change that

Reply:
This is all good, it just wasn't working for me because of [this issue](https://github.com/npm/cli/issues/1905) with the npm version I had on my laptop.